mkdocs.yml

```diff
@@ -85,6 +85,7 @@ nav:
     - Implementations:
         - List: implementations.md
         - Comparisons:
+          - v1.3: implementations/v1.3.md
           - v1.2: implementations/v1.2.md
           - v1.1: implementations/v1.1.md
           - v1.0: implementations/v1.0.md
```

Comment:
I think it would make sense to limit this to the trailing 3 versions of Gateway API, let's remove v1.0 as part of this.